velox/dwio/parquet/reader/ParquetReader.cpp

```diff
@@ -1166,4 +1167,37 @@ FileMetaDataPtr ParquetReader::fileMetaData() const {
   return readerBase_->fileMetaData();
 }
 
+const thrift::FileMetaData& ParquetReader::thriftFileMetaData() const {
+  return readerBase_->thriftFileMetaData();
+}
+
+void WriteBufferToFile(const std::shared_ptr<arrow::Buffer>& buffer, const std::string& file_path) {
+    auto source = std::make_shared<::arrow::io::BufferReader>(buffer);
```

Comment:
You simply write `buffer()`(returns a std::string_view) to a file using `LocalWriteFile`. See S3FileSystemTest.cpp for example.